website/src/pages/index.js

```diff
@@ -124,6 +147,7 @@ function Home() {
         </div>
       </header>
       <main>
+      <VideoContainer />
```

Comment:
Can the video be embedded after the features, so around line 164 below? I like how it looks on the Hermes page a little bit better.

Reply:
I am actually changing how it's going to be on Hermes page also to be above Features - it's better when into video comes before those, as many companies like Supabase do it that way also. We are changing to be like in https://docusaurus.io/

Reply:
The Supabase example is interesting because I feel that it's actually closer to how the infer landing page looks like now, with the "demo" video near the top ("Infer in action" subsection). The ELI5 video is awesome but, in my humble opinion, it's more of a "learn more" video than an intro video compared to the "infer in action" one. Curious to hear your thoughts on that. Anyhow, happy to land this as-is now and iterate further after that.

By the way, in the ELI5 video the "Infer in action" widget doesn't render, was there an issue with it? It should be showing https://asciinema.org/a/32101